helm-chart/renku-gateway/templates/configmap.yaml

```diff
@@ -79,10 +79,10 @@ data:
       [http.middlewares.development.headers]
         isDevelopment = true
       [http.middlewares.gitlab.AddPrefix]
-        prefix = "/gitlab/api/v4"
+        prefix = "{{ .Values.global.gitlab.servicePrefix }}/api/v4"
       [http.middlewares.jupyterhub.ReplacePathRegex]
         regex = "^/jupyterhub/(.*)"
-        replacement = "/jupyterhub/hub/api/$1"
+        replacement = "{{ .Values.global.jupyterhub.servicePrefix }}/api/$1"
```

Comment:
I think `/hub` is there by default

Reply:
> If you don't want a PR to be merged, make it a draft pr.

I would have, but you can't go back once you've pressed the wrong button...